org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -34,8 +35,16 @@
 	private DotnetNewWizardPage wizardPage;
 	@Override
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		// TODO
 		wizardPage = new DotnetNewWizardPage();
+
+		Object firstElement = selection.getFirstElement();
+		if (firstElement instanceof IResource) {
```

Comment:
Instead of just looking for the 1st element, it would be worth iterating on all elements and consider the 1st interesting one. So imagine someone selects 2 objects in the navigator and only the 2nd one adapts to an IResource but not the 1st one, we could have the wizard prepopulated anyway.